sormas-base/pom.xml

```diff
@@ -1168,6 +1191,14 @@
 			<name>Open Source Geospatial Foundation Repository</name>
 			<url>https://repo.osgeo.org/repository/release/</url>
 		</repository>
+		<repository>
+			<id>bintray</id>
+			<name>bintray</name>
+			<url>https://jcenter.bintray.com</url>
```

Comment:
Please add a comment that this repo is readOnly since 2021 and only listed for artifacts that may not be found somewhere else.